apps/mobile/src/screens/chat/chat.tsx

```diff
@@ -1,22 +1,22 @@
 import React from "react";
 
-import { Checkbox, ScreenWrapper, View } from "~/libs/components/components";
-import { useState } from "~/libs/hooks/hooks";
-import { globalStyles } from "~/libs/styles/styles";
+import { LoaderWrapper, ScreenWrapper } from "~/libs/components/components";
+import { DataStatus } from "~/libs/enums/enums";
+import { useAppSelector } from "~/libs/hooks/hooks";
+
+import { CategoriesForm } from "./components/categories-form";
 
 const Chat: React.FC = () => {
-	const [isChecked, setIsChecked] = useState<boolean>(false);
+	const { dataStatus } = useAppSelector((state) => state.quiz);
 
 	return (
-		<ScreenWrapper>
-			<View style={[globalStyles.flex1, globalStyles.gap12, globalStyles.p12]}>
-				<Checkbox
-					isChecked={isChecked}
-					label="Physical"
-					onValueChange={setIsChecked}
-				/>
-			</View>
-		</ScreenWrapper>
+		<LoaderWrapper isLoading={dataStatus === DataStatus.PENDING}>
+			<ScreenWrapper>
+				<ScreenWrapper>
+					<CategoriesForm />
+				</ScreenWrapper>
+			</ScreenWrapper>
```

Comment:
```suggestion
			<ScreenWrapper>
				<CategoriesForm />
			</ScreenWrapper>
```

Reply:
ScreenWrapper is to place UI in the safe area, there is no need for double wrapping.